jsonschema/golangci.next.jsonschema.json

```
@@ -2962,6 +2962,11 @@
               "type": "boolean",
               "default": false
             },
+            "msg-style": {
+              "description": "Enforce message style.",
+              "enum": ["", "lowercased", "capitalized"],
+              "default": false
```

```suggestion
              "default": ""
```

Shouldn't it be this?